pdk/1.0/modules/ROOT/pages/policies-pdk-troubleshooting.adoc

```diff
@@ -4,54 +4,115 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
-When using the Policy Development Kit (PDK), you might encounter issues with PDK itself. See the following section to help you resolve these issues:
+When using the Policy Development Kit (PDK), you might encounter issues with PDK itself. See the following sectios to help you resolve these issues:
 
-* <<cargo-credential-provider-configuration, Cargo Credential-Provider Configuration>>
+* <<cargo-generate>>
+* <<cli-credential>>
+* <<group-id-error>>
+* <<authorization-error>>
 
 To debug issues with your custom policy, see:
 
 * xref:policies-pdk-debug-local.adoc[]
 * xref:policies-pdk-debug-deployed-policies.adoc[]
 
+[[cargo-generate]]
+== Cargo Generate Error on a Mac Device
 
-== Cargo Credential-Provider Configuration
+When compiling Rust on a Mac Device for the first time, the following error might occur:
 
-When running the `setup` or `build` commands, you might receive the following error:
+[source,ssh]
+----
+$ cargo install cargo-generate
+error: linking with `cc` failed: exit status: 1
+[...]
+----
+
+To trouble shoot this error:
+
+. Ensure Xcode is installed.
++
+To install Xcode, execute the following command:
++
+[source,ssh]
+----
+xcode-select --install
+----
 
-[source,shell]
+. If Xcode is installed, create or edit the `~/.cargo/config.toml` by adding the following:
```

Comment:
If this is step 2 and we assume they've ensured Xcode is installed, can you remove up to the comma?